pages/Browse/components/SearchField/components/Suggestions/index.jsx

```diff
@@ -84,7 +85,7 @@ class SearchSuggestions extends Component {
     }
 
     return (
-      <div className={styles}>
+      <div className={[styles.wrapper, styles.bottom(this.props.bottomHeight)].join(' ')}>
```

Comment:
When the user starts searching after scrolling a little bit, the overlay gets visible between suggestions and keyboard. Additionally its possible to scroll the whole page by touching the section above the search field. That also doesn't look nice.

![bildschirmfoto 2018-07-06 um 17 46 38](https://user-images.githubusercontent.com/13087792/42388058-96d1d5da-8144-11e8-9509-da65cc60cc94.png)

Reply:
Done